src/network/networkd-manager.c

```diff
@@ -1000,6 +1119,31 @@ static int manager_save(Manager *m) {
         print_string_set(f, "DOMAINS=", search_domains);
         print_string_set(f, "ROUTE_DOMAINS=", route_domains);
 
+        SET_FOREACH(rule, m->rules, i) {
+                _cleanup_free_ char *from_str = NULL, *to_str = NULL;
+                fputs("RULE=", f);
+
+                if (!in_addr_is_null(rule->family, &rule->from)) {
+                        r = in_addr_to_string(rule->family, &rule->from, &from_str);
+                        if (r < 0)
+                                goto fail;
+
+                        fprintf(f, "from=%s%s/%hhu tos=%hhu fwmark=%"PRIu32"/%"PRIu32" table=%hhu", space ? " " : "", from_str,
+                                rule->from_prefixlen, rule->tos, rule->fwmark, rule->fwmask, rule->table);
```

Comment:
This will blow up if `from` and `to` are both specified — stuff like fwmark and table and whatnot will be printed twice.